src/cr/cube/measures/pairwise_pvalues.py

```diff
@@ -85,9 +91,20 @@ def _n_min(self):
         return min(self._slice.get_shape()) - 1
 
     @lazyproperty
-    def _numel(self):
-        """Number of elements of the dimension opposite to axis, as int."""
-        return self._slice.get_shape()[1 - self._axis]
+    def _off_margin(self):
```

Comment:
docstrings on all properties and methods please. I have no idea what this does or why. Don't make the (common) mistake of thinking that what is perfectly clear to you in this moment will be clear to others (or yourself next month :).

I'm not placing this same comment on every method without a docstring, but please look them all over and add one wherever missing, in this PR at least, but any others you come across along the way as well.

Reply:
It's perfectly clear to me :trollface: